Src/PChecker/CheckerCore/Coverage/ActivityCoverageReporter.cs

```diff
@@ -349,5 +353,15 @@ private static string GetMachineId(string nodeId)
 
         private static string GetStateId(string machineName, string stateName) =>
             string.Format("{0}::{1}", stateName, machineName);
+        
+        private static string GetSanitizedName(string name)
```

Comment:
Instead of doing this can you do `name.split(".").Last()`

Reply:
Done via [d752c52](https://github.com/p-org/P/pull/662/commits/d752c5233a417fd23f38ac20bb8b074c907881e7)